src/module/todo.js

```diff
@@ -0,0 +1,61 @@
+const ToDoList = JSON.parse(localStorage.getItem('ToDoList')) || [];
+const todoListElement = document.getElementById('todoList');
+const updateIndexValues = () => {
+  ToDoList.forEach((item, index) => {
+    item.index = index + 1;
+  });
+};
+
+const storeTasksToLocalStorage = () => {
+  localStorage.setItem('ToDoList', JSON.stringify(ToDoList));
+};
+const retrieveTasksFromLocalStorage = () => {
+  const storedTasks = localStorage.getItem('ToDoList');
+  return JSON.parse(storedTasks) || [];
+};
+
+const addTask = (description) => {
+  const newTask = {
+    description,
+    completed: false,
+    index: ToDoList.length + 1,
+  };
+  ToDoList.push(newTask);
+  storeTasksToLocalStorage();
+};
+
+const editTask = (description, index) => {
+  const taskToEdit = ToDoList.find((item) => item.index === index);
+  if (taskToEdit) {
+    taskToEdit.description = description;
+    storeTasksToLocalStorage();
+  }
+};
```

Comment:
[OPTIONAL] It would be best if you removed the red border highlights on the tasks after editing. The red border remains even after editing is done.

![image](https://github.com/melkamu12/To-Do-List/assets/38088818/2f64ec2b-b497-4ae2-875a-e9df71de5d58)